injector/src/main/java/edu/ucr/cs/riple/injector/Helper.java

```diff
@@ -131,6 +163,16 @@ private static void walk(Node cursor, List<Node> candidates, Predicate<Node> pre
         });
   }
 
+  /**
+   * Finds Top-Level type declaration within a {@link CompilationUnit} tree by name, this node is
+   * direct child of the compilation unit tree and can be: [{@link ClassOrInterfaceDeclaration},
```

Comment:
Missing article? Either "a direct child" or "the direct child"

Reply:
6079a72c488c2b651b698a6d88a0c7d29aaa784e